altair/vegalite/v5/schema/_config.py

```diff
@@ -0,0 +1,1054 @@
+# The contents of this file are automatically written by
+# tools/generate_schema_wrapper.py. Do not modify directly.
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Literal, Sequence, TypedDict
+
+# ruff: noqa: F405
+if TYPE_CHECKING:
+    from ._typing import *  # noqa: F403
+    from .core import *  # noqa: F403
+
+
+class RectConfigKwds(TypedDict, total=False):
+    """Placeholder doc."""
+
+    align: Align_T
+    angle: float
+    aria: bool
+    ariaRole: str
+    ariaRoleDescription: str
+    aspect: bool
+    baseline: str | Baseline_T
+    binSpacing: float
+    blend: Blend_T
+    color: str | LinearGradient | RadialGradient | ColorName_T
+    continuousBandSize: float
+    cornerRadius: float
+    cornerRadiusBottomLeft: float
+    cornerRadiusBottomRight: float
+    cornerRadiusTopLeft: float
+    cornerRadiusTopRight: float
+    cursor: Cursor_T
+    description: str
+    dir: TextDirection_T
+    discreteBandSize: float
+    dx: float
+    dy: float
+    ellipsis: str
+    endAngle: float
+    fill: str | None | LinearGradient | RadialGradient | ColorName_T
```

Comment:
TODO: `LinearGradient | RadialGradient`

Reply:
Technically representing `Gradient`, which is a `Union` of these types.

Currently `altair` defines these using an empty parent class:

https://github.com/vega/altair/blob/cfab6601aa649227c20880d05ae27799297694c3/altair/vegalite/v5/schema/core.py#L8039-L8045

Reply:
Fixed in [`e5c4b7f` (#3536)](https://github.com/vega/altair/pull/3536/commits/e5c4b7fb4e9bb55d51dd047229cd0f098f744a54)